apps/src/applab/applab.js

```diff
@@ -675,6 +617,15 @@ Applab.init = function (config) {
     studioApp.loadAudio(skin.failureSound, 'failure');
   };
 
+  config.shareWarningInfo = {
+    hasDataAPIs: function () {
+      return Applab.hasDataStoreAPIs(Applab.getCode());
+    },
+    onWarningsComplete: function () {
+      window.setTimeout(Applab.runButtonClick.bind(studioApp), 0);
```

Comment:
Super-obnoxious nit: I'd prefer `setTimeout` to `window.setTimeout` in general, since we can expect `setTimeout` to be on the global object in both browser and node environments, but `window` only exists in browser environments.  I realize that's dumb here since this code will always run in a browser/phantom, but I'm always looking for that stuff because I _want_ to be doing more testing in bare node.